README.md

```diff
@@ -1,73 +1,83 @@
-# sinch-java-sdk
+# Sinch for Java
+
 [![Maven Central](https://img.shields.io/maven-central/v/com.sinch.sdk/java-sdk.svg)](https://mvnrepository.com/artifact/com.sinch.sdk/java-sdk)
 
-## Documentation
-Sign up and find API documentation on the [Developer page](https://developers.sinch.com/)
+The Sinch java SDK helps you integrate with Sinch APIs from your java application. The latest version is released on [Maven Central](https://mvnrepository.com/artifact/com.sinch.sdk/java-sdk)
+
+## Quickstart - Send a message
+
+To complete the examples in this guide you will need a configured Sinch account with the [conversation API set up for SMS](https://developers.sinch.com/docs/conversation-getting-started). If you already have an account, continue with guide.
+
+In this guide, you will learn:
+
+1. Find necessary information to use the SDK.
+2. Install the Sinch for Java SDK
+3. Add and initialize the client
+4. Send a message.
 
 ## Installation
-The artifacts can be found on [Maven Central](https://mvnrepository.com/artifact/com.sinch.sdk/java-sdk).
 
-#### Gradle setup
+You find Sinch for Java on [Maven Central](https://mvnrepository.com/artifact/com.sinch.sdk/java-sdk). You can install it with either [Gradle](https://gradle.org/) or [Maven](https://maven.apache.org/)
+
+### Gradle setup
+
 ```groovy
 dependencies {
-    implementation "com.sinch.sdk:java-sdk:${sdkVersion}"
+    implementation "com.sinch.sdk:java-sdk:latest.release"
     ...
 }
 ```
 
-#### Maven setup
-```xml  
+### Maven setup
+
+```xml
 <dependencies>
     <dependency>
       <groupId>com.sinch.sdk</groupId>
       <artifactId>java-sdk</artifactId>
-      <version>${sdk.version}</version>
+      <version>LATEST</version>
```

Comment:
You will get warning while building with this version LATEST because it has been deprecated. Maven suggests to use specific version to have stability of the builds. Thats why we used property sdk.version

> dependencies.dependency.version' for com.sinch.sdk:java-sdk:jar is either LATEST or RELEASE (both of them are being deprecated) @ line 22, column 14

Reply:
if so it should alwasy be updated to the current version, or maybe have the wildcard 0.1.+ to get minor versions automatically?

Reply:
I would say this is a basic knowledge how to update versions of dependencies for someone who is using maven / gradle. So IMO what we had at the beginning was ok. But this is only my suggestion.

Reply:
I agree with Adam, if you think this really needs clarifying I would suggest a potential middle road being changing the 'variable' to be more descriptive, somehow pointing the user to the current version on maven that we display at the start of the readme.